frontend/plugins/pos_ui/src/modules/slot/components/slot.tsx

```
@@ -0,0 +1,870 @@
+      const currentY =
+        selectedNode.position?.y ?? selectedNode.data.positionY ?? 0;
+      if (formX !== currentX || formY !== currentY) {
+        updateNodePosition(selectedNode.id, { x: formX, y: formY }, true);
+      }
+    }
+  }, [slotDetail.left, slotDetail.top, selectedNode?.id, updateNodePosition]);
+
+  useEffect(() => {
+    const handleNodeResize = (event: Event) => {
+      const { detail } = event as CustomEvent;
+      if (detail && detail.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)